<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormMusicList</class>
 <widget class="QWidget" name="FormMusicList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>4</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelMusicText">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>410</y>
     <width>119</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Music</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonReturn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBluetooth">
   <property name="geometry">
    <rect>
     <x>1810</x>
     <y>1010</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMusicName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>770</y>
     <width>1901</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Resolution: 1920X1080</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelMusic">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>125</width>
     <height>125</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelThumbnail">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>270</y>
     <width>421</width>
     <height>341</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>GUIS/musicIcon.PNG</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="labelAirplay">
   <property name="geometry">
    <rect>
     <x>1670</x>
     <y>1010</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>150</y>
     <width>640</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(100, 100, 100);</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBottom">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>980</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>harman/kardon</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelWifi">
   <property name="geometry">
    <rect>
     <x>1740</x>
     <y>1010</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
